app/theme.ts

```diff
@@ -1,11 +1,12 @@
 import { extendTheme } from '@chakra-ui/react';
 
 const breakpoints = {
-    xs: '450px',
-    sm: '770px',
-    md: '1290px',
-    lg: '1500px',
-};
+    sm: "30em",
```

Comment:
double check these are good for all views

Reply:
ill be reverting this, i believe this is causing the problem